src/Operation/FindOne.php

```diff
@@ -126,4 +126,9 @@ public function execute(Server $server)
 
         return ($document === false) ? null : $document;
     }
+
+    public function getCommandDocument()
+    {
+        return $this->find->getCommandDocument();
```

Comment:
One other thing that just occurred to me. Unlike, `Find`, `FindAndModify` is an internal operation class (ideally, we don't want users to create it directly). I think the classes that compose `FindAndModify` (e.g. `FindOneAndUpdate`) should also implement Explainable and proxy the `getCommandDocument()` call, as you're doing here for `Find`.


 Similar to `Find` and `FindOne`, those sub-classes  various sub-classes of